<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManagerClass</class>
 <widget class="QMainWindow" name="FileManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
    <item>
     <widget class="QLineEdit" name="pathLineEdit">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
        </property>
        <property name="toolTip">
         <string>Копировать</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::EditCopy"/>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Удалить</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::EditDelete"/>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="listView">
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FileManager.qrc"/>
 </resources>
 <connections/>
</ui>
